### PR TITLE
Allow super user to simulate network (bug 1034118)

### PR DESCRIPTION
**Pull request by kumar303**

### CODE REVIEW

**Comment by kumar303** [comment]
@muffinresearch r? For the backend, see https://github.com/mozilla/webpay/pull/585

File: tests/helpers.js [hunk 1]
```diff
@@ -590,6 +616,18 @@ function sendEnterKey(selector) {
   }, selector || 'body');
 }
 
+
+function selectOption(selectContainer, optVal) {
+  // Select a form option.
+  // selectContainer: a CSS selector that gets the parent select tag.
```
Comment (muffinresearch) [inline tests/helpers.js]: This is probably more accurately the optionContainer?

File: public/js/models/transaction.js [hunk 2]
```diff
@@ -22,9 +22,17 @@ define([
...
     },
 
-    startTransaction: function() {
+    startTransaction: function(overrides) {
+      overrides = overrides || {};
       var options = {};
       var networkCodes = this.getNetworkCodes();
+      if (overrides.networkCodes) {
+        logger.warn('overriding network codes', networkCodes,
+                    'with:', overrides.networkCodes);
```
Comment (muffinresearch) [inline public/js/models/transaction.js]: Maybe stringify these objects?

**Comment by muffinresearch** [comment]
Nice tests, r+wc

File: public/js/models/transaction.js (hunk 2 above)
Comment (kumar303) [inline public/js/models/transaction.js]: oh yeah I forgot that adb logcat won't do that automatically

**Pull request merged**